css/_uswds-theme-custom-styles.scss

```diff
@@ -106,31 +108,59 @@ abbr[title="required"] {
 }
 
 .site-banner {
-  background-color: color("gray-90");
-
-  .usa-banner__inner {
-    max-width: 100%;
+  $site-banner-font-family: "lang";
+  $site-banner-icon-color: "blue-40v";
+  $site-banner-action-color: "gray-20";
+  @include typeset($site-banner-font-family);
+  @include at-media("tablet") {
+    font-size: font-size($site-banner-font-family, "3xs");
   }
+  background-color: color("ink");
 
-  .usa-banner__content {
-    color: color("white");
-    margin-left: 0;
+  .usa-accordion {
+    @include typeset($site-banner-font-family);
   }
 
   .usa-banner__button {
-    color: color("base-light");
-
+    color: color($site-banner-action-color);
+    font-size: font-size($site-banner-font-family, 1);
+    line-height: line-height($site-banner-font-family, 2);
     &:hover {
       color: color("white");
+      &::after {
+        background-color: color($site-banner-icon-color);
+      }
+    }
+    &::after {
+      background-color: color($site-banner-icon-color);
     }
   }
 
-  .usa-banner__header-text {
+  .usa-banner__content {
     color: color("white");
+    font-size: font-size($site-banner-font-family, 4);
+  }
+
+  .usa-banner__header {
+    font-size: font-size($site-banner-font-family, 1);
+  }
+
+  .usa-banner__header--expanded {
+    @include at-media("tablet") {
+      font-size: font-size($site-banner-font-family, 1);
+    }
   }
 
   .usa-banner__header-action {
-    color: color("gray-30");
+    color: color($site-banner-action-color);
+    line-height: line-height($site-banner-font-family, 2);
+  }
+
+  .usa-banner__header-text {
+    @include u-margin-y(0);
+    color: color("white");
+    font-size: font-size($site-banner-font-family, 1);
+    line-height: line-height($site-banner-font-family, 2);
```

Comment:
Added only the color and font customizations we actually use. Deleted other redundant banner styles